pysatNASA/instruments/templates/template_cdaweb_instrument.py

```diff
@@ -145,7 +180,7 @@ def default(self):
 
 # code should be defined below as needed
```

Comment:
Duplicate with line 164, may be unneeded.

Reply:
removed the prior line, since preprocess is still part of the standard bits

Reply:
added it back, since this is needed in both locations.  That will teach me not to look at the full code.